src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -128,7 +135,27 @@ export class OrganizationIntegrations extends AsyncComponent<
 
     const list = this.sortIntegrations(combined);
 
-    this.setState({list, displayedList: list}, () => this.trackPageViewed());
+    const allFeatureDescriptions: IntegrationFeature[] = list.reduce(
+      (acc, integration) => {
+        if (isSentryApp(integration)) {
+          const array = ['internal', 'unpublished'].includes(integration.status)
+            ? [{featureGate: integration.status, description: ''}]
+            : integration.featureData;
+          return acc.concat(...array);
+        }
+        if (isPlugin(integration)) {
+          return acc.concat(...integration.featureDescriptions);
+        }
+        return acc.concat(...integration.metadata.features);
+      },
+      [] as IntegrationFeature[]
```

Comment:
I think reduce takes a generic template, so you don't have to do this `reduce<IntegrationFeature[]>( ... )`